experimental/ast/type_generic.go

```diff
@@ -33,6 +33,10 @@ import (
 // that all generic types understood by your code are maps.
 //
 // TypeGeneric implements [Commas[TypeAny]] for accessing its arguments.
+//
+// # Grammar
+//
+//	TypeGeneric := TypePath `<` (Type, )* Type? `>`
```

Comment:
Similar to above about accepting empty brackets. I know allowing the comma to be optional exposes ambiguity for types (due to potential greediness of consuming prefixes), but I think we could still put that in the grammar and make that work in the parser, and just let it be greedy. (If being greedy is an issue, it's because there are multiple types without a comma, so we can still diagnose even if we are unconditionally greedy).
```suggestion
//	TypeGeneric := TypePath `<` ( Type `,`? )* `>`
```